src/parser/jobs/brd/modules/AdditionalStats.js

```diff
@@ -100,25 +102,55 @@ export default class AdditionalStats extends Module {
 			if (event.type.match(/^(apply|remove|refresh)(de)?buff(stack)?$/)) {
 				// In case it's a status on the selected player
 				if (event.targetID === this.combatants.selected.id && event.ability) {
-					this._player.statuses[event.ability.guid] = event.type.startsWith('apply') || event.type.startsWith('refresh')
+					const player = this._player
+					player.statuses[event.ability.guid] = {
+						isActive: event.type.startsWith('apply') || event.type.startsWith('refresh'),
+					}
+
+					// If the current status is a crit modifier, we add the strength onto it to use later
+					const critModifier = CRIT_MODIFIERS.find(cm => cm.id === event.ability.guid)
+					if (critModifier) {
+
+						// If it's a spear card, we get the modifier from the event, thank's to arcanum
+						if (critModifier.id === STATUSES.THE_SPEAR.id) {
+							player.statuses[event.ability.guid].strength = critModifier.strength * event.strengthModifier
+						// Otherwise, we get it from the const array
+						} else {
+							player.statuses[event.ability.guid].strength = critModifier.strength
+						}
+
+					}
 
 				// In case it's a status on an enemy, we attribute it to that specific enemy by the ID
 				} else if (!event.targetIsFriendly) {
 					const enemy = this._getEnemy(event.targetID)
-					enemy.statuses[event.ability.guid] = event.type.startsWith('apply') || event.type.startsWith('refresh')
+					enemy.statuses[event.ability.guid] = {
+						isActive: event.type.startsWith('apply') || event.type.startsWith('refresh'),
+					}
+					// If the current status is a crit modifier, we add the strength onto it to use later
+					const critModifier = CRIT_MODIFIERS.find(cm => cm.id === event.ability.guid)
+					if (critModifier) {
+
+						// If it's a spear card, we get the modifier from the event, thank's to arcanum
+						if (critModifier.id === STATUSES.THE_SPEAR.id) {
```

Comment:
Did you forget to get the strength?

Reply:
Did I? I'm confused. I'm gonna set as resolved and push my changes. Let me know if it's still there.